osbrain/tests/test_proxy.py

```diff
@@ -325,3 +329,13 @@ def shoot(self):
     assert not wayne._next_oneway
 
     assert wait_agent_attr(wayne, value=20 * ['bang!'], timeout=1.5)
+
+
+def test_agent_proxy_shutdown_connectionclosed():
```

Comment:
Missing test docstring. Even if just something like "ConnectionClosedError exceptions are allowed when shutting down a name server from a proxy".

Reply:
Or even better: "... when remotely shutting down the name server daemon from a proxy".

Or whatever you come up with. Just something a bit descriptive. :joy: